docs/install_molgenis/guide-standalone-server.md

```diff
@@ -0,0 +1,181 @@
+# Run MOLGENIS on a standalone server
+
+We support at the moment 2 OS variants.
+
+- CentOS (6.x)
+- RedHat (6.x)
+
+There are currently two ways of installing MOLGENIS on the server.
```

Comment:
I'd add a little picture to show the components so that folks have an idea of what they'll be installing.

Reply:
e.g. http://www.nomnoml.com/#view/%5BMOLGENIS%5D%20--%3E%20%5BPostgresql%5D%0A%5BMOLGENIS%5D%20--%3E%20%5BElasticSearch%5D%0A%5BMOLGENIS%5D%20--%3E%20%5BMinio%5D%0A%5BMOLGENIS%5D%20--%3E%20%5BOpenCPU%5D%0A%5BMOLGENIS%5D%20--%3E%20%5BPython%5D